docker-compose.yml

```diff
@@ -19,15 +19,26 @@ version: "3.8"
 
 
 services:
+  management:
+    build:
+      context: .
+      dockerfile: Dockerfile-management
+    depends_on:
+      - postgis
+      - tileserver
+    env_file: .env
+    volumes:
+      - .:/app
+
   webapp:
-    build: .
+    build:
+      context: .
+      dockerfile: Dockerfile-webapp
     command: venv/bin/gunicorn index:application -w 4 --bind 0.0.0.0:8080
     depends_on:
       - postgis
       - redis
     env_file: .env
-    #ports:
-    #  - 8080:8080  # for dev purpose
```

Comment:
127.0.0.1:8080:8080 should keep ports local without extra config files.

Reply:
it also will be available via `0.0.0.0` that I want to avoid, so will keep it as is

Reply:
No `127.0.0.1:8080:8080` specifically binds to local host interface.